static/ankersrv.js

```diff
@@ -105,7 +105,7 @@ $(function () {
         const data = JSON.parse(ev.data);
         if (data.commandType == 1001) {
             // Returns Print Details
-            $("#print-details").attr("style", "display: block;");
+            $("#print-details").removeClass("d-none");
```

Comment:
I was thinking about this, should we show this block of details all the time? There is no event that will hide it once it has been shown, outside of a refresh.

Reply:
Yeah, I was thinking the same thing while refreshing this. We don't have any good logic for keeping track of when to show this, and the user might even want to see the stats of the last print job after it's done.

I'll just make this always-on :+1: :white_check_mark: